code/modules/mod/mod.dm

```diff
@@ -0,0 +1,541 @@
+/// MODsuits, trade-off between armor and utility
+/obj/item/mod
```

Comment:
I think AnturK asked this as well, what's the point of this subtype? It looks like it doesn't actually implement anything, and is just a few variables.

Reply:
i found it really useful for spawning stuff during testing and i think it doesnt hurt? but i can change it when i finish the pr if you want

Reply:
Yeah, probably should.

I think you make an interesting call-out for spawning stuff, I think we should improve that experience specifically, but that's out of scope obviously.